src/redux/books/books.js

```diff
@@ -0,0 +1,35 @@
+const ADD_BOOKS = 'ADD_BOOKS';
+const REMOVE_BOOKS = 'REMOVE_BOOKS';
+
+const INITIAL_STATE = {
+  books: [],
+};
+
+const booksReducer = (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case ADD_BOOKS:
+      return {
+        ...state,
+        books: action.payload,
+      };
+    case REMOVE_BOOKS:
```

Comment:
- [ ] Please let's make sure that the particular book is removed from the `books` array in case the action type is `REMOVE_BOOKS`. You may use the ES6 filter() method to create a new array, which will not contain the book you want to remove from the store.